### DISCUSSION TITLE
Website not being developed on custom domain

### DISCUSSION THREAD

**Discussion by bajpai244** [vercel/community#33] [Help] [answered]
I have successfully changed my custom domain nameserver to vercel and the status is also showing verified, but when I visit the domain I get 404! , but the vercel domain is working fine

**Comment by mcsdevv** [comment, marked as answer]
Hello @bajpai244, have you added it to a project and if so, have you also added the "www" subdomain as well?

**Reply by bajpai244** [reply]
Hey!, thanks just added the www subdomain and now it is working, I actually tried it before but then it was not working, but now it works!

**Comment by rodrigoricky** [comment]
Same problem to me, and the answer above doesn't work.

**Reply by mcsdevv** [reply]
Hey @rodrigoricky, are you able to access the site correctly at the Vercel-generated deployment URL? Can you share this with us?

**Reply by rodrigoricky** [reply]
![image](https://user-images.githubusercontent.com/89636767/147927106-06e76c35-4347-46de-acfe-2b1d66edb44b.png)
It says this on the vercel dashboard, but when I try to access it, it says this 
![image](https://user-images.githubusercontent.com/89636767/147927131-4bc51f71-734e-4131-bb58-57146ffd628f.png)

**Reply by rodrigoricky** [reply]
but I can access the default vercel URL though

**Reply by mcsdevv** [reply]
Very odd, for me it redirects to the Vercel URL as expected, this could be DNS caching.
